samples/local.settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "SubscriptionId":  "<YOUR_SUBSCRIPTION_ID>",
+  "AVSResourceGroup":  "<YOUR_RESOURCE_GROUP>",
+  "AVSCloudName": "<YOUR_AVS_CLOUD>",
+  "CommandletId":  "Microsoft.AVS.Management/3.0.46/Get-CloudAdminGroups"
```

Comment:
Should probably change this one at least to be a little more recent (I know it'll just get out dated again but still)